<?xml version="1.0"?>
<launch>
  <node pkg="lidar_tracking"
    type="pcl_editor_node"
    name="pcl_editor_node"
    output="screen"
    respawn="true">
    <param name="file_path" value="$(find lidar_tracking)/params/"/>
    <param name="pcl_topic" value="/driver/hesai/pandar"/>
  </node>

  <node pkg="rqt_reconfigure"
    type="rqt_reconfigure"
    name="rqt_reconfigure"
    output="screen"
    respawn="true">
  </node>

  <node pkg="rviz"
  	type="rviz"
  	name="pcl_editor_rviz"
  	required="true"
  	args="-d $(find lidar_tracking)/params/pcl_editor.rviz" />
</launch>
